lib/LTP/utils.pm

```diff
@@ -87,6 +88,23 @@ sub get_ltp_version_file {
     return get_ltproot($want_32bit) . '/version';
 }
 
+sub log_versions {
+    my $kernel_pkg     = is_jeos ? 'kernel-default-base' : 'kernel-default';
```

Comment:
`kernel-default-base` now has its own set of QEM tests so this condition needs to be changed to `is_jeos || get_var('KERNEL_BASE')`